portal-frontend/src/app/features/create-application-dialog/create-application-dialog.component.scss

```diff
@@ -52,6 +52,22 @@
   display: none;
 }
 
+.step-description {
+  margin-bottom: 0.5rem;
```

Comment:
For css sizes in the Portal we are using the rem() function instead of rems directly, so this for example should be rem(8).